Makefile

```diff
@@ -0,0 +1,18 @@
+# Docker is the only dependency here
+# Build the dashboard image for host or cross-compile
+# This Makefile can only be run on a amd64 host
+
+build-image:
```

Comment:
All right, not problem. Just add a comment explaining why golang image was chosen.